service/index.js

```diff
@@ -0,0 +1,32 @@
+const Contact = require("./schemas/contacts");
+
+const getAllContacts = async () => Contact.find();
+
+const getContactById = async (contactId) => Contact.findById(contactId);
+
+const createContact = async ({ name, email, phone, favorite }) => {
+  return Contact.create({ name, email, phone, favorite });
+};
+
+const updateContact = async (contactId, fields) => {
+  return Contact.findByIdAndUpdate(contactId, fields, {
+    new: true,
+    strict: "throw",
+    runValidators: true,
+  });
+};
+
+const updateStatusContact = async (contactId, favorite) => {
```

Comment:
Tam gdzie używasz przesyłasz w drugim argumencie req.body, to nie wiadomo czy tu jest na pewno favorite co nie? Tak w funkcji nadrzędnej trzeba się upewnić że tak jest.

Reply:
coś takiego:
const updateStatusContact = async (contactId, favorite) => {
  const update = { favorite: favorite };
  return Contact.findByIdAndUpdate({ _id: contactId}, update, {new: true});
};